src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -32,6 +32,20 @@ spec:
       initContainers:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.volumes.sharedHome.nfsPermissionFixer.enabled }}
```

Comment:
While we're refactoring it, maybe we should call it sharedHome.initScript ?

Reply:
I'm not opposed to this, but I think the current naming is acceptable. It conveys intent but without exposing the details of how that intent is achieved which at this level I don't feel is important. Rewording things to include `initScript` is almost duplicating what we already know here. When looking at the surrounding YAML we see that this boolean is nested in an `initContainer` so I feel there is no need to double down on the name of this boolean.